bower.json

```diff
@@ -19,6 +19,7 @@
     "bower_components"
   ],
   "dependencies": {
-    "webcomponentsjs": "^0.7.22"
+    "webcomponentsjs": "^1.0.0",
+    "document-register-element": "^1.7.0"
```

Comment:
Because otherwise I'm getting "document.registerElement is not a function" in Firefox.